lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -5936,6 +5938,13 @@ void TypeChecker::checkConformancesInContext(IterableDeclContext *idc) {
   // Check all conformances.
   groupChecker.checkAllConformances();
 
+  // Check actor isolation.
+  for (auto *member : idc->getMembers()) {
+    if (auto *valueDecl = dyn_cast<ValueDecl>(member)) {
+      (void)getActorIsolation(valueDecl);
```

Comment:
This isn't checking the actor isolation of top-level declarations, but we should.

Reply:
Nice catch, fixed.